app/models/concerns/votes_counter.rb

```diff
@@ -20,8 +20,8 @@ def votes_count
   end
 
   def votes_alternative
-    votes.reduce(0) do |_ , v|
-      _ + v.value
+    votes.reduce(0) do |memo , vote|
+      memo + vote.value
```

Comment:
I agree with you that `_` & `v` is not very descriptive a variable name; but neither is `memo` :smile: